README.md

```
@@ -0,0 +1,213 @@
+npm run dev
+
+# フロントエンド（別ターミナル）
+cd frontend
+npm install
+cp .env.local.example .env.local  # 環境変数を設定
+npm run dev  # http://localhost:3002
+```
+
+## プロジェクト構成
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)